mlir/test/Dialect/Linalg/generalize-named-ops.mlir

```diff
@@ -29,6 +29,92 @@ func.func @generalize_matmul_buffer(%A : memref<16x8xf32>, %B: memref<8x32xf32>,
 
 // -----
 
+func.func @matmul_bcast_a(%arg0: memref<3xf32>, %arg1: memref<5x7xf32>, %arg2: memref<3x7xf32>) {
+  linalg.matmul indexing_maps = [
```

Comment:
Interesting. I think this should be an error. This is not a `linalg.matmul` at all right? I think at least one-contraction dimension is necessary. so `d2` should be there in LHS and RHS